lib/natalie/compiler/backends/cpp_backend.rb

```diff
@@ -285,6 +289,10 @@ def symbols_declaration
         "static SymbolObject *#{symbols_var_name}[#{@symbols.size}] = {};"
       end
 
+      def interned_strings_declaration
+        "static StringObject *#{interned_strings_var_name}[#{@interned_strings.size}] = {};"
```

Comment:
```suggestion
        "static StringObject *#{interned_strings_var_name}[#{@interned_strings.size}] = { 0 };"
```

Can we skip the later `memset()` if we zero initialize the array here?